src/MediaFileReader.js

```diff
@@ -219,6 +219,15 @@ class MediaFileReader {
 
     return size;
   }
+
+  getUnsignedInteger32At(offset: number): number {
```

Comment:
I found out that the same functionality is provided by getLong, so this new method has vanished entirely :)